Appium/Dockerfile

```diff
@@ -2,73 +2,69 @@ FROM ubuntu:16.04
 
 LABEL maintainer "Srinivasan Sekar <srinivasan.sekar1990@gmail.com>"
 
-WORKDIR /root
-
-ENV DEBIAN_FRONTEND=noninteractive \
-    DEBCONF_NONINTERACTIVE_SEEN=true
+ENV DEBIAN_FRONTEND=noninteractive
 
-#================================================================
-# Customize sources for apt-get
-#================================================================
-RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe\n" > /etc/apt/sources.list && \
-    echo "deb http://archive.ubuntu.com/ubuntu xenial-updates main universe\n" >> /etc/apt/sources.list && \
-    echo "deb http://security.ubuntu.com/ubuntu xenial-security main universe\n" >> /etc/apt/sources.list && \
+#=============
+# Set WORKDIR
+#=============
+WORKDIR /root
 
-#================================================================
-# Miscellaneous packages
-# Includes minimal runtime used for executing non GUI Java programs
-#================================================================
-    apt-get update -qqy && \
+#==================
+# General Packages
+#------------------
+# openjdk-8-jre-headless
+#   Java
+# ca-certificates
+#   SSL client
+# tzdata
+#   Timezone
+# unzip
+#   Unzip zip file
+# curl
+#   Transfer data from or to a server
+# wget
+#   Network downloader
+# libqt5webkit5
+#   Web content engine (Fix issue in Android)
+#==================
+RUN apt-get -qqy update && \
     apt-get -qqy --no-install-recommends install \
+    openjdk-8-jre-headless \
     ca-certificates \
-    openjdk-8-jre-headless && \
-    sed -i 's/securerandom.source=file:\/dev\/urandom/securerandom.source=file:\/dev\/.\/urandom/g' \
-       /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/java.security && \
-    sed -i 's/securerandom.source=file:\/dev\/random/securerandom.source=file:\/dev\/.\/urandom/g' \
-       /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/java.security && \
-    apt-get -qyy autoremove && \
-    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
-
-RUN apt-get update -qqy && \
-    apt-get -qqy --no-install-recommends install \
     tzdata \
+    unzip \
     curl \
-    xvfb \
-    x11vnc \
     wget \
     libqt5webkit5 \
-    unzip && \
-
-#=======================================
-# Setting JAVA_HOME in PATH
-#=======================================
-  { \
-		echo '#!/bin/sh'; \
-		echo 'set -e'; \
-		echo; \
-		echo 'dirname "$(dirname "$(readlink -f "$(which javac || which java)")")"'; \
-	} > /usr/local/bin/docker-java-home && \
-	chmod +x /usr/local/bin/docker-java-home && \
```

Comment:
this portion is required to set JAVA_HOME in PATH. This will break selendroid installation. Also for appium to run on android JAVA_HOME to be set properly in PATH.

Error from travis:
[0m[91mERR! [0m[91mSelendroid Could not find Java's 'jar' executable on your PATH. Please ensure it is present and try running install again